examples/flink-python-example/Dockerfile

```diff
@@ -36,7 +36,7 @@ apt-get clean && \
 rm -rf /var/lib/apt/lists/*
 
 # install PyFlink
-RUN pip3 install apache-flink==1.15.0
+RUN pip3 install "apache-flink>=1.15.0,<1.16.0"
```

Comment:
Is this necessary?

Reply:
Suggesting to be `RUN pip3 install "apache-flink>=1.15.0"`?.